clients/githubrepo/graphql.go

```diff
@@ -210,12 +211,18 @@ func pullRequestsFrom(data *graphqlData) []clients.PullRequest {
 func commitsFrom(data *graphqlData) []clients.Commit {
 	ret := make([]clients.Commit, 0)
 	for _, commit := range data.Repository.DefaultBranchRef.Target.Commit.History.Nodes {
+		var committer string
+		if commit.Committer.User.Login != nil {
+			committer = *commit.Committer.User.Login
+		} else if commit.Committer.Name != nil {
+			committer = *commit.Committer.Name
```

Comment:
this can be spoofed so we should not use it as committer, right?
Should it be 
```
} else{
  committer = "github"
}
```

as you suggested in https://github.com/ossf/scorecard/issues/1543#issuecomment-1023816104?

Reply:
> this can be spoofed so we should not use it as committer, right?

I didn't realize this can this be spoofed. How would someone do that? 

> as you suggested in [#1543 (comment)](https://github.com/ossf/scorecard/issues/1543#issuecomment-1023816104)?

Yes, this will be `github`. The `committer.name` is set to that value for commits which have `nil` User.

Reply:
I see. Maybe add a comment. Maybe we should also verify this is `github` in the code to validate our understanding, and catch any violating of our understanding.